lib/SymbolGraphGen/SymbolGraph.cpp

```diff
@@ -637,6 +628,19 @@ SymbolGraph::serializeDeclarationFragments(StringRef Key, Type T,
   T->print(Printer, Options);
 }
 
+namespace {
+
+const ValueDecl *getProtocolRequirement(const ValueDecl *VD) {
+  auto reqs = VD->getSatisfiedProtocolRequirements();
```

Comment:
mega nit, llvm code bases tend to do locals in `ThisKindOfCamelCase`

Reply:
This code's copy/pasted from `Symbol::getProtocolRequirement`, which (IIRC) was itself a refactoring of existing code. I can change it if you want, but i'd be touching some other places too.